tests/unit/utils.py

```diff
@@ -306,8 +306,9 @@ class BazCpdData(Data):
 
 
 class BazBucket(Container):
+    ROOT_NAME = 'root'
```

Comment:
For future proofing, I would just use the ROOT_NAME from ZarrIO (which I believe is already being imported). Similarly, in the docval, the default for 'name' can then be ROOT_NAME as imported from ZarrIO instead of redefining 'root' as a string.

Reply:
I don't see that we import ZarrIO to this file. This is also set in FooFile (like in HDMF), I will define it outside the class and use ROOT_NAME